Makefile.toml

```diff
@@ -15,15 +15,15 @@ cargo build --package gas-station --target wasm32-unknown-unknown --release --fe
 workspace = false
 clear = true
 script = """
-mkdir -p target/near/{gas_station,oracle,signer}
+mkdir -p target/near/{gas_station,local_ft,nft_key,oracle,signer}
```

Comment:
This should have been added already in #4.

Reply:
Yes, this is an older commit.